### PR TITLE
New article for Azure Stack HCI dashboard.

### PR DESCRIPTION
**Pull request by LouisBerner**

### CODE REVIEW

**Comment by Court72** [comment]
@ManikaDhiman 

After this PR is marked "Ready for review," can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Azure Stack HCI dashboard
```
Comment (alkohli) [inline azure-stack/hci/concepts/azure-stack-hci-dashboard.md:2]: Change to: Monitor at scale using the Azure Stack HCI overview page (preview)

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ms.topic: conceptual
+author: alkohli
+ms.subservice: azure-stack-hci
+ms.author: alkohli
+ms.reviewer: alkohli
+ms.date: 02/13/2024
+---
+
+# Azure Stack HCI dashboard
+
```
Comment (alkohli) [inline azure-stack/hci/concepts/azure-stack-hci-dashboard.md:13]: Change to: Monitor Azure Stack HCI systems using the dashboard (preview)

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+[!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
+
+This article details how to view and navigate your Azure Stack HCI dashboard in Azure portal. You can display the dashboard in chart view or in list view.
+
```
Comment (alkohli) [inline azure-stack/hci/concepts/azure-stack-hci-dashboard.md:17]: Change: This article details how to monitor your Azure Stack HCI systems using the dashboards in Azure portal. You can view the status of Azure Stack HCI systems as charts or lists.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+1. In Azure portal, go to **Azure Arc** > **Infrastructure** > **Azure Stack HCI** > **All clusters (preview)**.
+
+   All columns can be sorted by selecting a culmn header. You can filter the dashboard display by Subscription, Resource group, and Region. You can also group display results using the grouping control at top right.
+
+   :::image type="content" source="media/azure-stack-hci-dashboard/azure-stack-hci-dashboard-list-view.png" alt-text="Screenshot of the Azure Stack HCI dashboard list view." lightbox="media/azure-stack-hci-dashboard/azure-stack-hci-dashboard-list-view.png":::
+
+   Select a cluster name to view details about a cluster, or select a parameter to view details about cluster status.
+
+## Next steps
+
```
Comment (alkohli) [inline azure-stack/hci/concepts/azure-stack-hci-dashboard.md:57]: Add this new next step

- Learn about the various options available to monitor your Azure Stack HCI clusters in [What is Azure Stack HCI monitoring?](https://learn.microsoft.com/en-us/azure-stack/hci/concepts/monitoring-overview)

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- [Assess deployment readiness via the Environment Checker](../manage/use-environment-checker.md).
```
Comment (alkohli) [inline azure-stack/hci/concepts/azure-stack-hci-dashboard.md:58]: Remove all the existing next steps as these are not relevant.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+---
+title: Azure Stack HCI dashboard
+description: Learn to view and navigate your Azure Stack HCI dashboard in Azure portal.
+ms.topic: conceptual
```
Comment (alkohli) [inline azure-stack/hci/concepts/azure-stack-hci-dashboard.md:4]: Monitor at scale Azure Stack HCI systems using the overview page in the portal (preview)

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+## View the dashboard chart view
+
+The Azure Stack HCI dashboard displays overview and detailed information about your clusters.
+
+To view the dashboard chart view:
+
+1. In Azure portal, go to **Azure Arc** > **Infrastructure** > **Azure Stack HCI** > **Overview (preview)**.
+
+   You can filter the dashboard display by Subscription, Resource group, and Region.
+
```
Comment (alkohli) [inline azure-stack/hci/concepts/azure-stack-hci-dashboard.md:27]: Change to: You can filter the dashboard display by **Subscription**, **Resource group**, and **Region**.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md (hunk 3 above)
Comment (alkohli) [reply azure-stack/hci/concepts/azure-stack-hci-dashboard.md:17]: You also need to add the preview disclaimer include here.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md (hunk 1 above)
Comment (LouisBerner) [reply azure-stack/hci/concepts/azure-stack-hci-dashboard.md:2]: Done > updated the title.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md (hunk 2 above)
Comment (LouisBerner) [reply azure-stack/hci/concepts/azure-stack-hci-dashboard.md:13]: Done > Updated the H1.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md (hunk 3 above)
Comment (LouisBerner) [reply azure-stack/hci/concepts/azure-stack-hci-dashboard.md:17]: Done > updated the description in text and in metadata.

Done > added include: [!INCLUDE [hci-preview](../../includes/hci-preview.md)].

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md (hunk 4 above)
Comment (LouisBerner) [reply azure-stack/hci/concepts/azure-stack-hci-dashboard.md:57]: Done > added new link.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md (hunk 5 above)
Comment (LouisBerner) [reply azure-stack/hci/concepts/azure-stack-hci-dashboard.md:58]: Done > removed links.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md (hunk 6 above)
Comment (LouisBerner) [reply azure-stack/hci/concepts/azure-stack-hci-dashboard.md:4]: Done > updated.

File: azure-stack/hci/concepts/azure-stack-hci-dashboard.md (hunk 7 above)
Comment (LouisBerner) [reply azure-stack/hci/concepts/azure-stack-hci-dashboard.md:27]: Done > revised to bold.

**Pull request merged**